packages/react-learn-card/package.json

```diff
@@ -70,14 +71,13 @@
   "dependencies": {
     "@learncard/core": "workspace:*",
     "date-fns": "^2.28.0",
+    "fitty": "^2.3.6",
     "numeral": "^2.0.6",
+    "pretty-bytes": "^6.0.0",
     "qrcode.react": "^3.0.2",
-    "react": "^18.2.0",
+    "react": "link:~/WeLibraryOS/wl-react/node_modules/react",
     "react-barcode": "^1.4.5",
-    "react-dom": "^18.2.0"
-  },
-  "peerDependencies": {
```

Comment:
hmm im going to do a couple of tests, but it seems like we should move react and react dom to peer dependencies and perhaps dev dependencies but remove from dependencies, will try linking this locally from learncardapp.

Reply:
Delete react and react-dom from dependencies (they're still in peerDeps). Storybook works fine, but not really if that has other implications.. @TaylorBeeston?

Reply:
@smurflo2 Cool! I just did a test where i linked this locally in the learn-card-app repo and it worked. I think it's fine but to be safe you can add it to dev-dependencies (so react/react dom are in peerdependencies and devdepencies but not dependencies) - that seems pretty safe IMO.